astro/src/components/Aside.astro

```diff
@@ -36,6 +36,10 @@ const icons: Record<NonNullable<Props['type']>, { viewBox: string; d: string }>
 	version: {
 		viewBox: '0 0 16 16',
 		d: 'M7.75 0a.75.75 0 0 1 .75.75V3h3.634c.414 0 .814.147 1.13.414l2.07 1.75a1.75 1.75 0 0 1 0 2.672l-2.07 1.75a1.75 1.75 0 0 1-1.13.414H8.5v5.25a.75.75 0 0 1-1.5 0V10H2.75A1.75 1.75 0 0 1 1 8.25v-3.5C1 3.784 1.784 3 2.75 3H7V.75A.75.75 0 0 1 7.75 0Zm4.384 8.5a.25.25 0 0 0 .161-.06l2.07-1.75a.248.248 0 0 0 0-.38l-2.07-1.75a.25.25 0 0 0-.161-.06H2.75a.25.25 0 0 0-.25.25v3.5c0 .138.112.25.25.25h9.384Z',
+	},
+	theme: {
```

Comment:
Do these changes need to be run by Sean?

Reply:
I think so. Yellow with a paintbrush doesn't tell me to stop, and people need to listen to this one. So I think it needs to be red with a stop sign or some such, but I would go to sean and see what he thinks about the UX here.

Honestly we need to start breaking out action items from release notes because I am not sure that this is the best way to let people know that they need manual intervention.

Reply:
I don't think we go all the way to red on this page. Database migrations only gets an orange.

What if every bullet point was a big warning text box? Then everyone would stop and read every one of them very carefully.

We definitely need a better way to manage release notes.